pallets/anchors/src/lib.rs

```diff
@@ -100,25 +106,26 @@ pub mod pallet {
 	{
 		/// Type representing the weight of this pallet
 		type WeightInfo: WeightInfo;
+
+		/// Currency as viewed from this pallet
+		type Currency: ReservableCurrency<Self::AccountId>;
+
+		/// Amount of funds reserved in a [`Pallet::pre_commit()`] call.
+		/// These funds will be unreserved once the user make the [`commit()`] succesfully
+		/// or call [`Pallet::evict_pre_commits()`]
+		#[pallet::constant]
+		type PreCommitDeposit: Get<BalanceOf<Self>>;
```

Comment:
Probably this type can be removed once `pallet_fees` be updated with a storage key-enum based.

Reply:
@mikiquantum Do we want to use a key from the pallet key for the pre-commit deposit?

Reply:
Yes, I would say so, since we already have the setup for it.